static/src/javascripts/es6/projects/common/modules/crosswords/helpers.js

```diff
@@ -46,6 +90,15 @@ const cellsForEntry = (entry) => isAcross(entry) ?
         y: y
     }));
 
+const cellsForClue = (entries, clue) => {
```

Comment:
I think I'm doing the same in [my PR](https://github.com/guardian/frontend/pull/10541/files#diff-54d92cdd4bc5edacc9c0286b23c2fbeeR497) :) You've nicely abstracted it into a helper though, so maybe I can refactor once you've merged! I don't think we're clashing so shouldn't cause any issues both merging in, but maybe whoever merges second can merge in from master and check..?